kinto/plugins/default_bucket/__init__.py

```diff
@@ -4,15 +4,13 @@
 from pyramid import httpexceptions
 from pyramid.settings import asbool
 from pyramid.security import NO_PERMISSION_REQUIRED, Authenticated
-
 from cliquet.errors import raise_invalid
 from cliquet.utils import build_request, reapply_cors, hmac_digest
 from cliquet.storage import exceptions as storage_exceptions
-
 from kinto.authorization import RouteFactory
 from kinto.views.buckets import Bucket
 from kinto.views.collections import Collection
-
+from pyramid import route_path
```

Comment:
Actually it seems that you cannot import ``route_path``, but that you should be using ``request.route_path``